### ISSUE TITLE
[Assembly Instructions] - Solder the encoders before the switches top cover

### ISSUE THREAD

**Issue by carvallegro** [joshajohnson/Hub16#11]
I find it's easier to add the rotary encoders to the PCB before the top plate as the pins can be a bit fiddly and you can adjust them easily as you insert it. The top plate slides over the encoders quite easily (but on't if you put the knobs on). 

You can solder the pins after to be safe as the encoders are quite steady

Maybe the documentation could be updated to reflect that, happy to open a PR to make the change.

**Comment by joshajohnson** [comment]
Thanks for the feedback! 

I'm planning to update the assembly instructions soon as I've just updated the PCB / case design and the assembly instructions are quite different, so will make sure to switch things up for that. Just waiting on the production PCBs to arrive which all going well will be within the fortnight.

I've also moved the encoder cutouts to make life a bit easier even if you assemble it the other way around - it's been a common complaint that they are hard to fit.
